components/prize_pool/wikis/apexlegends/prize_pool_custom.lua

```diff
@@ -0,0 +1,70 @@
+---
+-- @Liquipedia
+-- wiki=apexlegends
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Opponent = require('Module:Opponent')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {8, 4, 2}
+local TYPE_MODIFIER = {Online = 0.65}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	local prizePool = PrizePool(args):create()
+
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
+	return prizePool:build()
+end
+
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	lpdbData.weight = CustomPrizePool.calculateWeight(
+		lpdbData.prizemoney,
+		Variables.varDefault('tournament_liquipediatier'),
+		placement.placeStart,
+		Variables.varDefault('tournament_type')
+	)
+
+	lpdbData.extradata.series = Variables.varDefault('tournament_series', '')
+	lpdbData.extradata.liquipediatiertype2 = Variables.varDefault('tournament_tiertype2', '')
+	lpdbData.extradata.notabilitymod = Variables.varDefault('tournament_notability_mod', '1')
+
+	local participantLower = mw.ustring.lower(lpdbData.participant)
+
+	Variables.varDefine(participantLower .. '_prizepoints', lpdbData.extradata.prizepoints)
+	Variables.varDefine('enddate_'.. participantLower, lpdbData.date)
+
+	if Opponent.isTbd(opponent.opponentData) then
+		Variables.varDefine('minimum_secured', lpdbData.extradata.prizepoints)
+	end
+
+	return lpdbData
+end
+
+function CustomPrizePool.calculateWeight(prizeMoney, tier, place, type)
+	if String.isEmpty(tier) then
+		return 0
+	end
+
+	local tierValue = TIER_VALUE[tier] or TIER_VALUE[tonumber(tier) or ''] or 1
+
+	return tierValue + tierValue * prizeMoney * (TYPE_MODIFIER[type] or 1) / place
+end
```

Comment:
Doesn't have the same logic as the old one
https://liquipedia.net/apexlegends/Module:Weight

Reply:
How would I have to go about changing the logic to match the one on Module:Weight?

Reply:
you could just require Module:Weight and use its functionality